en/guide/using-template-engines.md

```diff
@@ -36,21 +26,21 @@ $ npm install pug --save
 ```
 
 <div class="doc-box doc-notice" markdown="1">
-Express-compliant template engines such as Jade and Pug export a function named `__express(filePath, options, callback)`,
-which is called by the `res.render()` function to render the template code.
+Express-compliant template engines such as Pug export a function named `__express(filePath, options, callback)`,
+which `res.render()` calls to render the template code.
 
 Some template engines do not follow this convention. The [Consolidate.js](https://www.npmjs.org/package/consolidate)
```

Comment:
The author of `consolidate.js` says that in the future the package will stop receiving updates in favor of `@ladjs/consolidate`. It might be a good idea to update the link.

Reply:
Updated